editor/script_editor_debugger.cpp

```diff
@@ -645,7 +645,11 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 							Ref<Script> script(var);
 							if (!script.is_null()) {
 								ScriptInstance *script_instance = script->placeholder_instance_create(debugObj);
-								debugObj->set_script_and_instance(var, script_instance);
+
+                                // fix: VisualScript resource
+                                if (script_instance) {
+                                    debugObj->set_script_and_instance(var, script_instance);
+                                }
```

Comment:
It is a violation of Code style guidelines (https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html). Use a clang-format tool to prevent this.

You should use tabulations instead of spaces here.